packages/financial-templates-lib/src/price-feed/CreatePriceFeed.js

```diff
@@ -107,28 +107,29 @@ async function createPriceFeed(logger, web3, networker, getTime, config) {
       config.priceFeedDecimals, // This defaults to 18 unless supplied by user
       uniswapBlockCache
     );
-  } else if (config.type === "defipulsetvl") {
-    const requiredFields = ["lookback", "minTimeBetweenUpdates", "defipulseApiKey"];
+  } else if (config.type === "defipulse") {
+    const requiredFields = ["lookback", "minTimeBetweenUpdates", "apiKey", "project"];
 
     if (isMissingField(config, requiredFields, logger)) {
       return null;
     }
 
     logger.debug({
       at: "createPriceFeed",
-      message: "Creating DefiPulseTotalPriceFeed",
+      message: "Creating DefiPulsePriceFeed",
       config
     });
 
-    return new DefiPulseTotalPriceFeed(
+    return new DefiPulsePriceFeed(
       logger,
       web3,
-      config.defipulseApiKey,
```

Comment:
You should leave this named `defipulseApiKey`. The reason is this [issue](https://github.com/UMAprotocol/protocol/issues/2588). If the `apiKey` has the same variable name for multiple price feeds, then it can make it tricky for liquidator bot runners to not overload the variable

Reply:
Done